### PR TITLE
Privacy quickstart 

### PR DESCRIPTION
**Pull request by MadelineMurray**
Privacy quickstart

### CODE REVIEW

File: docs/Privacy/Privacy-Quickstart.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+description: Pantheon private network quickstart tutorial
```
Comment (NicolasMassart) [inline docs/Privacy/Privacy-Quickstart.md]: may we also add the privacy word in the meta description ?

Comment (MadelineMurray) [reply docs/Privacy/Privacy-Quickstart.md]: Done

File: docs/Privacy/Privacy-Quickstart.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+The Private Network with Privacy Enabled Quickstart runs a private network of Pantheon and Orion nodes managed by Docker Compose.
+It is an expanded version of the [Private Network Quickstart](../Tutorials/Private-Network-Quickstart.md). 
+
+You can use the [Block Explorer](../Tutorials/Private-Network-Quickstart.md#block-explorer), 
+make [JSON-RPC requests](../Tutorials/Private-Network-Quickstart.md#run-json-rpc-requests), and 
+create [transactions using Metamask](../Tutorials/Private-Network-Quickstart.md#creating-a-transaction-using-metamask)
+as described in the [Private Network Quickstart tutorial](../Tutorials/Private-Network-Quickstart.md). 
+This tutorial describes how to use the examples provided in the EEAJS library to [create and send private transactions](#send-private-transactions-and-read-values). 
+
+!!! important
```
Comment (NicolasMassart) [inline docs/Privacy/Privacy-Quickstart.md]: Perhaps we could add this to the other quickstart too (another PR)

File: docs/Privacy/Privacy-Quickstart.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+```
+
+!!! tip 
+    The network takes a minute or so to get started.  If you get a ` Error: socket hang up` error, the network
+    isn't fully setup. Wait and then run the command again. 
+
+The Event Emitter logs are displayed. 
+
+```bash
+
```
Comment (NicolasMassart) [inline docs/Privacy/Privacy-Quickstart.md]: Something missing here ?

File: docs/Privacy/Privacy-Quickstart.md (hunk 2 above)
Comment (MadelineMurray) [reply docs/Privacy/Privacy-Quickstart.md]: Already done :-)

File: docs/Privacy/Privacy-Quickstart.md (hunk 3 above)
Comment (MadelineMurray) [reply docs/Privacy/Privacy-Quickstart.md]: Yep - have added it now

**Pull request merged**